packages/near-membrane-dom/package.json

```diff
@@ -21,13 +21,13 @@
     "exports": {
         ".": {
             "types": "./types/index.d.ts",
-            "import": "./dist/index.mjs",
-            "require": "./dist/index.js"
+            "import": "./dist/index.js",
+            "require": "./dist/index.cjs.js"
         },
         "./*.js": {
-            "import": "./dist/*.mjs",
-            "require": "./dist/*.js",
-            "default": "./dist/*.js"
+            "import": "./dist/*.js",
```

Comment:
☝️ the issue here is that *.js matches *.cjs.js

Reply:
What about `*.cjs.js` and `*.mjs.js`

Reply:
Let's try it!